persistence/test/application_test.go

```diff
@@ -246,6 +236,16 @@ func TestGetSetStakeAmount(t *testing.T) {
 	require.Equal(t, newStakeAmount, stakeAmountAfter, "unexpected status")
 }
 
+func TestGetAllApps(t *testing.T) {
+	db := NewTestPostgresContext(t, 0)
+
+	updateApp := func(db *persistence.PostgresContext, app *genesis.App) error {
+		return db.UpdateApp(app.Address, "https://olshansky.info", app.MaxRelays, []string{"OLSH"})
```

Comment:
NIT: Can make these constants and global var
`https://olshansky.info`
`[]string{"OLSH"}`

Reply:
Any objections to naming it `OlshanskyURL` and `OlshanskyChain`

Reply:
Lol no objection